lib/cc/test_coverage_report.rb

```diff
@@ -25,7 +25,7 @@ def run
           coverage = test_file_report.fetch("attributes").fetch("coverage")
 
           lines = patch.changed_line_numbers.map do |line_number|
-            covered = !coverage[line_number - 1].nil? && coverage[line_number - 1].nonzero?
+            covered = !coverage[line_number - 1].nil? && !coverage[line_number - 1].zero?
```

Comment:
Funny thing is that Code Climate suggested I use #nonzero? instead of `> 0` originally. This works for me though.